revup/cherry_pick.py

```diff
@@ -13,17 +12,27 @@ async def main(args: argparse.Namespace, git_ctx: git.Git) -> int:
     """
     branch_to_pick = args.branch[0]
     remote_branch_to_pick = git_ctx.ensure_branch_prefix(branch_to_pick)
-    branch_exists, remote_branch_exists = await asyncio.gather(
-        git_ctx.is_branch_or_commit(branch_to_pick),
-        git_ctx.is_branch_or_commit(remote_branch_to_pick),
-    )
-    if remote_branch_exists and not branch_exists:
+    branch_exists = await git_ctx.is_branch_or_commit(branch_to_pick)
+
+    if not branch_exists:
         logging.warning(
```

Comment:
these can probably just be info instead of warning now

Reply:
Done

Reply:
Will let you hit the merge button